typescript/infra/scripts/check/check-warp-deploy.ts

```diff
@@ -40,29 +42,33 @@ async function main() {
     WarpRouteIds.ArbitrumBaseBlastBscEthereumGnosisLiskMantleModeOptimismPolygonScrollZeroNetworkZoraMainnet,
   ];
 
+  const registries = [DEFAULT_REGISTRY_URI, DEFAULT_GITHUB_REGISTRY];
+  const warpCoreConfigMap = await getWarpConfigMapFromMergedRegistry(
+    registries,
+  );
```

Comment:
Using `getWarpConfigMapFromMergedRegistry()` and then filtering in memory is better than fetching one at a time by warpRouteId because we want to reduce rate limiting